modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/add_to_cart/data.js

```diff
@@ -138,10 +138,12 @@ export async function addToCart(
 			includedCartItem &&
 			!Liferay.CommerceContext.showSeparateOrderItems
 		) {
-			includedCartItem.quantity = Number(
-				Number(includedCartItem.quantity + cpInstance.quantity).toFixed(
-					cpInstance.skuUnitOfMeasure?.precision || 0
-				)
+			includedCartItem.quantity =
+				parseFloat(includedCartItem.quantity) +
+				parseFloat(cpInstance.quantity);
+
+			includedCartItem.quantity = includedCartItem.quantity.toFixed(
```

Comment:
I think we need to add the Number(***) because otherwise the quantity becomes a string (the toFixed method returns a string)